navigator/frontend/webpack.config.js

```diff
@@ -52,8 +52,13 @@ const APP_NAME = 'Navigator';
  */
 module.exports = (env) => {
   const paths_case_check = env && env.paths_case_check  || 'true';
-  const in_dir           = env && env.bazel_in_dir  || __dirname;
-  const out_dir          = env && env.bazel_out_dir || path.join(__dirname, 'dist');
+  // const in_dir           = env && env.bazel_in_dir  || __dirname;
```

Comment:
Could you add a comment pointing to [this issue](https://github.com/bazelbuild/rules_nodejs/issues/1554) for future reference?

Reply:
added a comment.